README.md

```diff
@@ -1,12 +1,14 @@
 ![Ripple](/images/ripple.png)
 
+**Do you work at a digital asset exchange or wallet provider?** 
+
+Please [contact us](mailto:support@ripple.com). We can help guide your integration.
+
 # What is Ripple?
-Ripple is a network of computers which use the [Ripple consensus algorithm](https://www.youtube.com/watch?v=pj1QVb1vlC0) to atomically settle and record
-transactions on a secure distributed database, the Ripple Consensus Ledger
-(RCL). Because of its distributed nature, the RCL offers transaction immutability
-without a central operator. The RCL contains a built-in currency exchange and its
-path-finding algorithm finds competitive exchange rates across order books
-and currency pairs.
+Ripple connects banks, payment providers and digital asset exchanges via RippleNet to provide one frictionless experience to send money globally.
+
+# What is the XRP Ledger?
+The XRP Ledger is a network of computers, which uses [a consensus algorithm](https://www.youtube.com/watch?v=pj1QVb1vlC0) to atomically settle and record transactions on a secure distributed database. Because of its distributed nature, the XRP Ledger offers transaction immutability without a central operator. The XRP Ledger contains a built-in currency exchange and its path-finding algorithm finds competitive exchange rates across order books and currency pairs.
 
 ### Key Features
```

Comment:
Seems odd not to mention Interledger compatibility when that's one of our other big advantages. Escrow and PaymentChannels provide key compatibility making it easy for Interledger payments to pass through the XRP Ledger.